server/region_handler_test.go

```diff
@@ -195,3 +195,80 @@ func (ts *TidbRegionHandlerTestSuite) stopServer(c *C) {
 		ts.server.Close()
 	}
 }
+
+func (ts *TidbRegionHandlerTestSuite) prepareData(c *C) {
+	db, err := sql.Open("mysql", dsn)
+	c.Assert(err, IsNil, Commentf("Error connecting"))
+	defer db.Close()
+	dbt := &DBTest{c, db}
+
+	dbt.mustExec("create database tidb;")
+	dbt.mustExec("use tidb;")
+	dbt.mustExec("create table tidb.test (a int auto_increment primary key, b int);")
+	dbt.mustExec("insert tidb.test values (1, 1);")
+	txn1, err := dbt.db.Begin()
+	c.Assert(err, IsNil)
+	_, err = txn1.Exec("update tidb.test set b = b + 1 where a = 1;")
+	c.Assert(err, IsNil)
+	_, err = txn1.Exec("insert tidb.test values (2,2);")
+	c.Assert(err, IsNil)
+	err = txn1.Commit()
+	c.Assert(err, IsNil)
+}
+
+func (ts *TidbRegionHandlerTestSuite) TestGetMvcc(c *C) {
+	ts.startServer(c)
+	ts.prepareData(c)
+	defer ts.stopServer(c)
+	resp, err := http.Get(fmt.Sprintf("http://127.0.0.1:10090/mvcc/key/tidb/test/1"))
```

Comment:
It's referenced on [tidb_test.go](https://github.com/pingcap/tidb/blob/master/server/tidb_test.go#L43). And I'm not pretty sure that whether we should keep same with it. @siddontang